docs/audits/keyboard-audit.md

```diff
@@ -0,0 +1,77 @@
+# Notebook 7 Keyboard Navigation Audit
```

Comment:
Does it really make sense to duplicate so much information? Most of the content in this Markdown file is coming from https://github.com/jupyter/notebook/issues/6595 isn't it?

Reply:
Agree, at most; this doc should be a mini TLD;R and link directly to the audit sources. We want to avoid content duplication as much as possible.